src/js/gi/components/profile/Calculator.jsx

```diff
@@ -16,34 +20,39 @@ const EligibilityDetails = ({ expanded, toggle }) => (
   </div>
 );
 
-const CalculatorInputs = ({ expanded, toggle }) => (
+const CalculatorInputs = ({ expanded, toggle, inputs, displayedInputs, onInputChange, onShowModal }) => (
   <div className="calculator-inputs">
     <button onClick={toggle} className="usa-button-outline">
       {expanded ? 'Hide' : 'Edit'} calculator fields
     </button>
     <div className="form-expanding-group-open">
-      {expanded ? <CalculatorForm/> : null}
+      {expanded ? <CalculatorForm
+          inputs={inputs}
+          displayedInputs={displayedInputs}
```

Comment:
Will combine these at some point similar to how `calculatedBenefits.outputs` is formatted.